concepticondata/conceptlists/Saxena-2013-157.tsv

```diff
@@ -0,0 +1,158 @@
+Saxena-2013-157-31	31	autumn	1276	AUTUMN
+Saxena-2013-157-32	32	bird	937	BIRD
+Saxena-2013-157-33	33	blood	946	BLOOD
+Saxena-2013-157-34	34	bone	1394	BONE
+Saxena-2013-157-35	35	butter	1245	BUTTER
+Saxena-2013-157-36	36	cat (f,m)	1208	CAT
+Saxena-2013-157-37	37	child	2099	CHILD
+Saxena-2013-157-38	38	cloud	1489	CLOUD
+Saxena-2013-157-39	39	copper	1825	COPPER
+Saxena-2013-157-40	40	day	1225	DAY (NOT NIGHT)
+Saxena-2013-157-41	41	dog	2009	DOG
+Saxena-2013-157-42	42	egg	744	EGG
+Saxena-2013-157-43	43	fire	221	FIRE
+Saxena-2013-157-44	44	food	256	FOOD
+Saxena-2013-157-45	45	forest	420	FOREST
+Saxena-2013-157-46	46	fruit	1507	FRUIT
+Saxena-2013-157-47	47	glacier		
+Saxena-2013-157-48	48	goat	1502	GOAT
+Saxena-2013-157-49	49	gold	1369	GOLD
+Saxena-2013-157-50	50	grass	606	GRASS
+Saxena-2013-157-51	51	hen	1514	HEN
+Saxena-2013-157-52	52	iron	621	IRON
+Saxena-2013-157-53	53	lamb	820	LAMB
+Saxena-2013-157-54	54	leaf	628	LEAF
+Saxena-2013-157-55	55	man (human)	683	PERSON
+Saxena-2013-157-56	56	man (male)	1554	MAN
+Saxena-2013-157-57	57	meat	2615	FLESH OR MEAT
+Saxena-2013-157-58	58	milk	635	MILK
+Saxena-2013-157-59	59	moon	1313	MOON
+Saxena-2013-157-60	60	mountain	639	MOUNTAIN
+Saxena-2013-157-61	61	night	1233	NIGHT
+Saxena-2013-157-62	62	pond; lake	3321	POND OR LAKE
+Saxena-2013-157-63	63	rain	2108	RAINING OR RAIN
+Saxena-2013-157-64	64	river	666	RIVER
+Saxena-2013-157-65	65	root	670	ROOT
+Saxena-2013-157-66	66	rope	1218	ROPE
+Saxena-2013-157-67	67	salt	1274	SALT
+Saxena-2013-157-68	68	seed	714	SEED
+Saxena-2013-157-69	69	sheep	1331	SHEEP
+Saxena-2013-157-70	70	silver	759	SILVER
+Saxena-2013-157-71	71	sky	1732	SKY
+Saxena-2013-157-72	72	snake	730	SNAKE
+Saxena-2013-157-73	73	snow	784	SNOW
+Saxena-2013-157-74	74	spring (season)	1537	SPRINGTIME
+Saxena-2013-157-75	75	star	1430	STAR
+Saxena-2013-157-76	76	stone	857	STONE
+Saxena-2013-157-77	77	sugar	862	SUGAR
+Saxena-2013-157-78	78	summer	1342	SUMMER
+Saxena-2013-157-79	79	sun	1343	SUN
+Saxena-2013-157-80	80	tree	906	TREE
+Saxena-2013-157-81	81	village	930	VILLAGE
+Saxena-2013-157-82	82	water	948	WATER
+Saxena-2013-157-83	83	wind	960	WIND
+Saxena-2013-157-84	84	winter	1471	WINTER
+Saxena-2013-157-85	85	woman	962	WOMAN
+Saxena-2013-157-86	86	yak	2477	YAK
+Saxena-2013-157-87	87	yak (female)	2479	FEMALE YAK
+Saxena-2013-157-88	88	year	1226	YEAR
+Saxena-2013-157-89	89	bad	1292	BAD
+Saxena-2013-157-90	90	beautiful	1427	BEAUTIFUL
+Saxena-2013-157-91	91	big	1202	BIG
+Saxena-2013-157-92	92	black	163	BLACK
+Saxena-2013-157-93	93	cold	1287	COLD
+Saxena-2013-157-94	94	dry	1398	DRY
+Saxena-2013-157-95	95	good	1035	GOOD
+Saxena-2013-157-96	96	green	1425	GREEN
+Saxena-2013-157-97	97	long	1203	LONG
+Saxena-2013-157-98	98	new	1231	NEW
+Saxena-2013-157-99	99	old	1229	OLD
+Saxena-2013-157-100	100	red	156	RED
+Saxena-2013-157-101	101	round	1395	ROUND
+Saxena-2013-157-102	102	small	1246	SMALL
+Saxena-2013-157-103	103	straight	1404	STRAIGHT
+Saxena-2013-157-104	104	wet	1726	WET
+Saxena-2013-157-105	105	warm	2272	HOT OR WARM
```

Comment:
Similar to _flesh/meat_, therefore, I would map it to 1232 WARM.

Reply:
Likewise.